api/server/volume.go

```diff
@@ -774,6 +774,11 @@ func (vd *volAPI) enumerate(w http.ResponseWriter, r *http.Request) {
 		vol, err := volumes.Inspect(ctx, &api.SdkVolumeInspectRequest{VolumeId: s})
 		if err == nil {
 			vols = append(vols, vol.GetVolume())
+		} else if sdk.IsErrorNotFound(err) {
```

Comment:
should we use this in snapEnumerate too?

Reply:
Yeah, i'll update it.